content/pages/2024/tickets.md

```diff
@@ -0,0 +1,53 @@
+---
+title: EuroSciPy 2024 - Tickets
+url: 2024/tickets.html
+save_as: 2024/tickets.html
+section: euroscipy_2024
+template: page
+slug: tickets_2024
+---
+
+# Tickets for EuroSciPy
+
+EuroSciPy 2024 will be held from August 26th to August 30th. It will be both
+in person and virtual. In person, it will be held at Maritime University of 
+Szczecin, Szczecin, Poland. Details of the virtual conference will be
+published soon.
```

Comment:
I would be safe for the moment and not mention anything about the online tickets since we might not have the possibility to stream and only record.

```suggestion
Szczecin, Szczecin, Poland.
```